### PR TITLE
added SpigotMC 1.15.1

### PR DESCRIPTION
**Pull request by vk2gpz**
added the latest (1.15.1) SpigotMC java doc.

### CODE REVIEW

**Comment by Kapeli** [comment]
Multiple authors are not supported. In the JSON the author needs to be an object, not an array.

File: docsets/SpigotMC/docset.json [hunk 1]
```diff
@@ -1,17 +1,21 @@
 {
     "name": "SpigotMC",
-    "version": "1.12.2",
+    "version": "1.15.1",
     "archive": "SpigotMC.tgz",
```
Comment (Kapeli) [inline docsets/SpigotMC/docset.json]: You need to update the archive at this path as well.

Comment (vk2gpz) [reply docsets/SpigotMC/docset.json]: what do you mean by "update the archive"?  The latest SpigotMC.tgz is in 1.15.1 version folder.

Comment (Kapeli) [reply docsets/SpigotMC/docset.json]: I mean the archive at this path: `docsets/SpigotMC/SpigotMC.tgz`. The `specific_version` docset archive and the main docset archive are separate.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**